im.pidgin.Pidgin3.yml

```diff
@@ -0,0 +1,126 @@
+app-id: im.pidgin.Pidgin3
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: pidgin3
+finish-args:
+  # X11 + XShm access
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  # Wayland access
+  - "--socket=wayland"
+  # GPU acceleration if needed
+  - "--device=dri"
+  # Needs to talk to the network:
+  - "--share=network"
+  # Audio
+  - "--socket=pulseaudio"
+  # Notifications (purple-toast)
+  - "--talk-name=org.freedesktop.Notifications"
```

Comment:
This is handled magically by the toolkit. Are you sure you need it?

Reply:
Do you mean the runtime is already requesting this? We use `GNotification` from GLib that might have a portal, but I'm not sure. Likewise, we're also using libcanberra to talk to pulse so I think we need that too?

Reply:
gnotification should use the portal automatically afaik.